### ISSUE TITLE
Device hub caught a fatal exception

### ISSUE THREAD

**Issue by Rojoyinc** [ASCOMInitiative/ASCOMDeviceHub#5]
I go to run device hub and get errors. I've reinstalled. I've rebooted... still persists
I see this error. 

Device hub caught a fatal exception during startup 
Input string was not in a correct format.

**Comment by astroman133** [comment]
What version of DH are you running? If it won't come up at all, then look at the number in the name of the installer file.

Also, something changed in the DH configuration; like the Dome Azimuth Adjustment slider being on a .5 degree setting. This is fixed in the latest release on Github.

Any idea of other changes? You can view the settings in the ASCOM Profile Explorer for Telescope, Dome, and Focuser.

**Comment by Rojoyinc** [comment]
I reinstalled latest yesterday and like before "a reinstall" fixes it for a while. It worked last night but now again tonight I'm getting same error.   Version, I can't see since it won't run, but I'll install it again and let you know.

WHAT SUCKS is each time I install, I lose my setup data and I have to go into slaving settings each time I reinstall.   ZIP doesn't say - but unzipped in the Inno Setup/output dir the setup file is called  ASCOM.DeviceHub(6.4.1.1)_setup exe

YEP I reinstalled it again and it now runs. (but doesn't have my settings) sigh

IT use to work fine.  Thanks for your work!  Better than poth. .
